torch/_inductor/codecache.py

```diff
@@ -2508,6 +2508,7 @@ def __init__(
         lib_path: str,
     ):
         self.lib_path = lib_path
+        self.is_open = False
```

Comment:
If we don't put that here, and the subsequent call to cdll.LoadLibrary fails for any reason, an endless recursion results..